war/src/main/js/widgets/config/tabbar.less

```diff
@@ -94,233 +94,253 @@
   }
 
 }
-#jenkins{
-  .jenkins-config {
-    border:1px solid @solid-border;
-    padding:10px;
-    border-top:none;
-    background:@bright;
-    .border-radius-bottom(@radius);
-
-    .showTabs{
-      display:block;
-      text-align:right;
 
+// #jenkins{
+//     table.config-table {
+//       .section-header-row {
+//         background: none;
+//         border-color: transparent
+//       }
+//       .section-header {
+//         margin-top: 10px;
+//       }
+//     }
+
+//     .config-table {
+
+//       .section-header{
+//         margin: 30px 0px 5px -10px;
+//         font-size: 1.25rem;
+//         padding: 10px 15px;
+//         border: none;
+//         border-top: 1px solid #ddd;
+//       }
+//       .attach-previous {
+//         margin-left:3px;
+//       }
+//       .validation-error-area td{
+//         padding:0 5px;
+//       }
+
+//       .stapler-class-bag td {
+//         padding: 0px;
+//       }
+
+//       td.setting-name{
+//         line-height:29px;
+//       }
+//       td.setting-main{
+//         input[type='radio'],
+//         input[type='checkbox']{
+//           position:relative;
+//           top:.33em;
+//         }
+//       }
+//       td{
+//         padding:5px;
+
+//         textarea.setting-input {
+//           display:block;
+//         }
+
+//       }
+//       td[colspan="4"]:empty{
+//         display:none
+//       }
+//       td.setting-help{
+//         vertical-align:top;
+//         padding:5px 10px 5px 5px
+//       }
+//       td.setting-leftspace:after {
+//         content:' ';
+//         display:block;
+//         width:13px;
+//         height:1px;
+//       }
+//     }
+//   }
+// }
```

Comment:
Commented out instead of not added why?

Reply:
TBH I haven't gotten around doing proper CSS cleanup, ought to do it soon.

Reply:
I've just removed it feel free to do more cleanup on top